pom.xml

```diff
@@ -537,6 +537,18 @@
             </dependency>
 
             <dependency>
+                <groupId>org.openjdk.jmh</groupId>
+                <artifactId>jmh-core</artifactId>
+                <version>1.14.1</version>
```

Comment:
Yes, we should remove it and let it inherit it from the parent POM. It might require some changes to the code due to some backward-incompatible changes in JMH a few versions ago.